src/wkb.jl

```diff
@@ -219,7 +224,7 @@ function GI.getgeom(
     geom::WKBtype,
     i::Integer,
 )
-    ncoord = GI.ncoord(geom)
+    ncoord = GI.ncoord(T, geom)
     GI.getgeom(T, geom, i, ncoord)
```

Comment:
Should this be an internal method rather than adding non-standard methods to `GI.getgeom` ?

Reply:
Ah, not part of this PR, but I should fix that yes. It seems I already thought of your suggestion to pass ncoord form the toplevel in a hacky way.